rules/gluster-volume.libsonnet

```diff
@@ -6,10 +6,16 @@
         rules: [
           {
             expr: |||
-               sum(max(gluster_subvol_capacity_used_bytes{%(glusterExporterSelector)s}) BY (volume, subvolume)) BY (volume)
+              sum(max(gluster_subvol_capacity_used_bytes{%(glusterExporterSelector)s}) BY (volume, subvolume)) BY (volume)
             ||| % $._config,
             record: 'gluster:volume_capacity_used_bytes_total:sum',
           },
+          {
+            expr: |||
+              sum(max(gluster_subvol_capacity_total_bytes{%(glusterExporterSelector)s}) BY (volume, subvolume)) BY (volume)
+            ||| % $._config,
+            record: 'gluster:volume_capacity_available_bytes_total:sum',
```

Comment:
gluster:volume_capacity_total_bytes:sum

Reply:
Agreed. Capacity available might be perceived as total - used.

Reply:
Ack. Will update